README.md

```diff
@@ -1520,6 +1520,7 @@ Update Time, five active automations, webhooks.
   * [Calendar Icons Generator](https://calendariconsgenerator.app/) -- Generate an entire year's worth of unique icons in a single click, absolutely FREE
   * [Image BG Blurer](https://imagebgblurer.com/) -- Generate a blurred background frame for an image, using that image source as the background blur, for Notion, Trello, Jira, and more tools
   * [Webstudio](https://webstudio.is/) -- Open-source alternative to Webflow. The free plan offers unlimited websites on their domain. Five websites with custom domains. Ten thousand page views/month. 2 GB asset storage.
+  * [EarlyBird](https://earlybird.im) -- No-code landing page builder designed for product idea validation. The free plan includes one landing page and up to 1,000 visits per month.
```

Comment:
100 visits a month it says, the paid plan comes with 1000/month. thats just not useful so we can remove this one

<img width="758" alt="Screenshot 2023-11-30 at 16 17 44" src="https://github.com/ripienaar/free-for-dev/assets/82342/4360fa83-53c8-49a0-bf45-9803c97ad530">